web/less/bubble.less

```diff
@@ -262,6 +262,9 @@
   .bubble-content {
     border-radius: 10px;
     box-shadow: -1px 1px 4px rgba(0, 0, 0, 0.2);
+    height: 50vh;
+    overflow-y: scroll;
+    max-height: 412px;
```

Comment:
I made this feature responsive to respond to all devices. when there is adequate height this bubble will take full height of 412px. This is calculated based on the height of all children in their original height in px.

let me know if you have a use-case where the element is cropped and is not scrollable.

Reply:
Yes, but it will never be taller than 50vh, which may be smaller than 412px, even on a view port that has adequate height to display the entire menu.

Reply:
For example, this is the appearance on a 780-pixel tall view port (the menu is cropped):

![image](https://user-images.githubusercontent.com/726379/94484653-a11f4980-0191-11eb-8443-2984334e06c3.png)

If I remove the `50vh` in this case, the full menu is displayed:

![image](https://user-images.githubusercontent.com/726379/94484728-be541800-0191-11eb-86e4-b341a2ff5072.png)